### PR TITLE
test(react): adding high contrast visual regression tests - 2

### PR DESCRIPTION
**Pull request by MI6-286**
## Summary of the changes
Adding high contrast visual regression tests for IcButton & IcChip.

### CODE REVIEW

File: packages/react/cypress-image-diff-screenshots/baseline/IcChip.cy.tsx-custom-colour.png
Comment (GCHQ-Developer-299) [inline packages/react/cypress-image-diff-screenshots/baseline/IcChip.cy.tsx-custom-colour.png:1]: Seems the dark blue chip colour from the original screenshot has been lost - intentional?

Comment (MI6-286) [reply packages/react/cypress-image-diff-screenshots/baseline/IcChip.cy.tsx-custom-colour.png:1]: Yeah, changed it as I felt the second custom colour is testing the same thing (though a different colour). But equally happy to add back in the dark blue colour.

Comment (GCHQ-Developer-299) [reply packages/react/cypress-image-diff-screenshots/baseline/IcChip.cy.tsx-custom-colour.png:1]: Gotcha! Currently it looks like the second and third pill are identical now - I think the return of the dark blue colour (especially now there's an icon) would be good, please :)

Comment (MI6-286) [reply packages/react/cypress-image-diff-screenshots/baseline/IcChip.cy.tsx-custom-colour.png:1]: No problem!

**Review by gd2910 — APPROVED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Pull request merged**